adoc/retail_chap_admin.adoc

```diff
@@ -199,7 +198,60 @@ Only use colons (``:``), hyphens (``-``) or underscores (``_``) in hardware type
 Spaces and other non-alphanumeric characters will be removed when the name is processed.
 ====
 +
-. TODO: Apply hardware type
+. . In the {susemgr} {webui}, navigate to menu:Systems[Overview], and click on the [guimenu]``Groups`` tab.
+. Click the [btn]``+ Create new`` button.
```

Comment:
I think we must not quote button label decoration such as a leading + sign (+ icon).

Reply:
Ooh, that's a nasty habit from my last job!